### ISSUE TITLE
Test failures with Ruby2.7

### ISSUE THREAD

**Issue by utkarsh2102** [marcandre/packable#11]
Hi,

There are some test failures with Ruby2.7:
```
┌──────────────────────────────────────────────────────────────────────────────┐
│ Run tests for ruby2.7 from debian/ruby-test-files.yaml                       │
└──────────────────────────────────────────────────────────────────────────────┘

RUBYLIB=/home/utkarsh/debian/ruby-team/packale/ruby-packable/debian/ruby-packable/usr/lib/ruby/vendor_ruby:. GEM_PATH=/home/utkarsh/debian/ruby-team/packale/ruby-packable/debian/ruby-packable/usr/share/rubygems-integration/all:/home/utkarsh/.gem/ruby/2.7.0:/var/lib/gems/2.7.0:/usr/lib/ruby/gems/2.7.0:/usr/share/rubygems-integration/2.7.0:/usr/share/rubygems-integration/all:/usr/lib/x86_64-linux-gnu/rubygems-integration/2.7.0 ruby2.7 -ryaml -e YAML.load_file\(\"debian/ruby-test-files.yaml\"\).each\ \{\ \|f\|\ require\ f\ \}
Traceback (most recent call last):
	8: from /usr/lib/ruby/vendor_ruby/minitest.rb:68:in `block in autorun'
	7: from /usr/lib/ruby/vendor_ruby/minitest.rb:139:in `run'
	6: from /usr/lib/ruby/vendor_ruby/minitest.rb:847:in `start'
	5: from /usr/lib/ruby/vendor_ruby/minitest.rb:847:in `each'
	4: from /usr/lib/ruby/vendor_ruby/minitest.rb:757:in `start'
	3: from /usr/lib/ruby/vendor_ruby/minitest.rb:757:in `puts'
	2: from /home/utkarsh/debian/ruby-team/packale/ruby-packable/debian/ruby-packable/usr/lib/ruby/vendor_ruby/packable/extensions/io.rb:62:in `write_with_packing'
	1: from /home/utkarsh/debian/ruby-team/packale/ruby-packable/debian/ruby-packable/usr/lib/ruby/vendor_ruby/packable/extensions/io.rb:86:in `pack_and_write'
/home/utkarsh/debian/ruby-team/packale/ruby-packable/debian/ruby-packable/usr/lib/ruby/vendor_ruby/packable/extensions/io.rb:86:in `pos': Illegal seek @ rb_io_tell - <STDOUT> (Errno::ESPIPE)
ERROR: Test "ruby2.7" failed. Exiting.
```

Any clue about the fix?

**Comment by pocheptsov** [comment]
Pull request [#12](https://github.com/marcandre/packable/pull/12) should fix it.

**Comment by marcandre** [comment]
Should be fixed in 1.3.12.

Thanks

**Comment by pocheptsov** [comment]
Thank you Marc, appreciate it.
